tests/e2e/concatenate-xpath-strings.wdio-spec.js

```diff
@@ -0,0 +1,44 @@
+const concatenateStrings = require('../page-objects/forms/concatenate-strings.wdio.page');
+const common = require('../page-objects/common/common.wdio.page');
+const utils = require('../utils');
+const constants = require('../constants');
+const genericForm = require('../page-objects/forms/generic-form.wdio.page');
+const loginPage = require('../page-objects/login/login.wdio.page');
+const { expect } = require('chai');
+
+
+const userContactDoc = {
+  _id: constants.USER_CONTACT_ID,
+  name: 'Jack',
+  date_of_birth: '',
+  phone: '+64274444444',
+  alternate_phone: '',
+  notes: '',
+  type: 'person',
+  reported_date: 1478469976421,
+  parent: {
+    _id: 'some_parent',
+  },
+};
+
+// If this test starts failing then we need to document in the release notes that we've removed the deprecated
+// feature allowing for concatenation of strings
+describe('Concatenate xpath strings', () => {
+  before(() => loginPage.cookieLogin());
+  beforeEach(() => concatenateStrings.configureForm(userContactDoc));
+  afterEach(() => utils.revertDb([], true));
```

Comment:
I believe this is not needed anymore. We already have a global `after` cleanup and there's just one test suite here.